### ISSUE TITLE
Test zoom images upload pls

### ISSUE THREAD

**Issue by mk-hasan** [mk-hasan/SISRIA#1]
Please upload the zoom image test set... if images are not enough for testing, i can make more test iamge set ..no problem. have fun

**Comment by ghost** [comment]
u can check if the first pair of image could be used and the others will be uploaded soon koz the lagging...

**Comment by ghost** [comment]
em...in the dataset_new file

**Comment by mk-hasan** [comment]
Hey thanks for the pics. i think i need to take few more photos after consulting with Andy. As all the photos are not low resolution after cropping :p . We need to show those pictures to andy.

**Comment by ghost** [comment]
5 pairs uploaded. Yes im not sure if it is required koz of the resolution that's why i ask u to check if it can be used and the pics which u took actually will exist some "shake" differences, so the  alignment might get some deviation

**Comment by ghost** [comment]
yeah i know the resolution prob. if it cant be used i can crop it again and it will be low resolution
